src/navigations/root-stack.js

```diff
@@ -5,66 +5,64 @@ import PropTypes from 'prop-types';
+import SignedChatScreen from '../screens/ChatScreen/SignedChatScreen';
 import CreatePost from '../screens/CreatePost';
 import DiscoveryScreenV2 from '../screens/DiscoveryScreenV2';
 import DomainScreen from '../screens/DomainScreen';
 import FollowersScreen from '../screens/Followings/FollowersScreen';
 import FollowingScreen from '../screens/Followings/FollowingScreen';
-import HelpCenter from '../screens/WebView/HelpCenter';
-import HomeBottomTabs from './HomeBottomTabs';
 import ImageViewerScreen from '../screens/ImageViewer';
-import KeyboardWrapper from './KeyboardWrapper';
+import ChooseUsername from '../screens/InputUsername';
 import LinkContextScreen from '../screens/LinkContextScreen';
 import LocalCommunity from '../screens/LocalCommunity';
-import NetworkStatusIndicator from '../components/NetworkStatusIndicator';
-import OneSignalNavigator from './OneSignalNavigator';
 import OtherProfile from '../screens/OtherProfile';
 import OtherProfilePostDetail from '../screens/OtherProfilePostDetail';
 import OtherProfileReplyComment from '../screens/OtherProfileReplyComment';
 import PostDetailPage from '../screens/PostPageDetail';
-import PrivacyPolicies from '../screens/WebView/PrivacyPolicies';
 import ProfilePostDetail from '../screens/ProfilePostDetail';
 import ProfileReplyComment from '../screens/ProfileReplyComment';
 import ReplyComment from '../screens/ReplyComment';
 import Settings from '../screens/Settings';
 import SignIn from '../screens/SignInV2';
-import SignedChatScreen from '../screens/ChatScreen/SignedChatScreen';
 import TopicMemberScreen from '../screens/TopicMemberScreen';
 import TopicPageScreen from '../screens/TopicPageScreen';
 import Topics from '../screens/Topics';
 import VideoViewerScreen from '../screens/VideoViewer';
+import HelpCenter from '../screens/WebView/HelpCenter';
+import PrivacyPolicies from '../screens/WebView/PrivacyPolicies';
 import WhotoFollow from '../screens/WhotoFollow';
 import api from '../service/config';
-import {
-  AddParticipant,
-  ChannelScreen,
-  ChatDetailPage,
-  ContactScreen,
-  DetailDomainScreen,
-  DetailGroupImage,
-  GroupInfo,
-  GroupMedia,
-  GroupSetting,
-  ProfileScreen
-} from '../screens';
 import {InitialStartupAtom, LoadingStartupContext} from '../service/initialStartup';
 import {NavigationConstants} from '../utils/constants';
-import {followersOrFollowingAtom} from '../screens/ChannelListScreen/model/followersOrFollowingAtom';
-import {useInitialStartup} from '../hooks/useInitialStartup';
+import HomeBottomTabs from './HomeBottomTabs';
+import KeyboardWrapper from './KeyboardWrapper';
+import OneSignalNavigator from './OneSignalNavigator';
 
 const RootStack = createNativeStackNavigator();
 
 export const RootNavigator = () => {
   const initialStartup = useRecoilValue(InitialStartupAtom);
   const [following, setFollowing] = useRecoilState(followersOrFollowingAtom);
   const loadingStartup = useInitialStartup();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `useSafeAreaInsets` from the import statements and its usage in the component suggests that you are no longer considering safe area insets in your layout. This could potentially lead to UI issues on devices with notches or rounded corners where some parts of your app's UI might be hidden or obstructed. If this is intentional and you've handled these cases differently, then it's fine. Otherwise, consider reintroducing the use of `useSafeAreaInsets`.

```diff
- import {useSafeAreaInsets} from 'react-native-safe-area-context';
+ // No replacement needed if safe area handling is done differently

...

- const insets = useSafeAreaInsets();
+ // No replacement needed if safe area handling is done differently
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->